src/SFA.DAS.EAS.Employer_Account.Database/Tables/TransferConnectionInvitation.sql

```diff
@@ -7,6 +7,10 @@
 	[DeletedBySender] BIT NOT NULL,
 	[DeletedByReceiver] BIT NOT NULL,
 	[CreatedDate] DATETIME NOT NULL,
+	[ConnectionHash] AS (CASE	
+							WHEN ReceiverAccountId > SenderAccountId THEN Convert(varchar(10),SenderAccountId) + '/' + Convert(varchar(10),ReceiverAccountId) 
```

Comment:
Mix of upper and lowercase T-SQL keywords.

Reply:
Done